cider/type/plan/Expr.h

```diff
@@ -181,20 +181,27 @@ class Expr : public std::enable_shared_from_this<Expr> {
 
   JITExprValue* get_expr_value() const { return expr_var_.get(); }
 
+  void set_null_datas(cider::jitlib::JITValuePointer& val) { null_vals_.push_back(val); }
```

Comment:
use `set_nulls` / `get_nulls` instead and rename `null_vals_` to `nulls_` to keep all definition consistent.

Reply:
FIX